Formula/sslh.rb

```diff
@@ -20,16 +13,16 @@ class Sslh < Formula
     sha256 "2a712be56b116244717fb4e414846b6b9373bf2b0482b764b2142096cffbac18" => :mountain_lion
   end
 
-  head "https://github.com/yrutschle/sslh.git"
-
   depends_on "libconfig"
 
   def install
-    ENV.j1
-    system "make", "install", "PREFIX=#{prefix}"
+    ENV.deparallelize
+    system "make", "install", "PREFIX=#{prefix}", "BINDIR=#{bin}"
+
+    prefix.install "README.MacOSX"
```

Comment:
FWIW, this file literally only tells people to install `sslh` with MacPorts and offers up an example `launchctl` plist. We can write the latter into the formula ourselves.